<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>991</width>
      <height>771</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="videoLayout">
       <item>
        <widget class="QLabel" name="videoText">
         <property name="text">
          <string>视频源</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="videoSource">
         <property name="text">
          <string>请输入视频地址</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="chooseText">
         <property name="text">
          <string>选择算法</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="algorithmList">
         <item>
          <property name="text">
           <string>请选择算法</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="serviceLayout">
       <item>
        <widget class="QLabel" name="serviceUrlText">
         <property name="text">
          <string>算法服务地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="serviceUrl">
         <property name="text">
          <string>请输入算法服务器地址</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="accountLayout">
         <item>
          <widget class="QLineEdit" name="username">
           <property name="text">
            <string>账号</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="password">
           <property name="text">
            <string>密码</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="video">
       <property name="text">
        <string>视频预览区</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonsLayout">
       <item>
        <widget class="QPushButton" name="openVideo">
         <property name="text">
          <string>启动视频流</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeVideo">
         <property name="text">
          <string>关闭视频流</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startAnalyze">
         <property name="text">
          <string>启动算法分析</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopAnalyze">
         <property name="text">
          <string>停止算法分析</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="description">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>算法测试v0.1 版权@新航物联网 2022 by QILIN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
